parlai/core/image_featurizers.py

```diff
@@ -39,6 +46,7 @@
     'resnext101_32x16d_wsl_spatial': ['resnext101_32x16d_wsl', -2],
     'resnext101_32x32d_wsl_spatial': ['resnext101_32x32d_wsl', -2],
     'resnext101_32x48d_wsl_spatial': ['resnext101_32x48d_wsl', -2],
+    'faster_r_cnn_152_32x8d': ['', -1],
```

Comment:
Why is this string empty?

Reply:
this is a mapping from ParlAI `--image-mode` to the PyTorch cnn instantiation string & layer to use. This dict is where we get our image mode choices from, so i've included the faster r cnn features in here, however the extractor does not use a PyTorch CNN like the other image modes do (it has its own extractor) so there's no relevant mapping